core/src/main/java/org/projectnessie/cel/interpreter/Activation.java

```diff
@@ -32,7 +33,7 @@ public interface Activation {
    * ResolveName returns a value from the activation by qualified name, or false if the name could
    * not be found.
    */
-  Object resolveName(String name);
+  Pair<Object, Boolean> resolveName(String name);
```

Comment:
This is very go-idiomatic but does not feel quite right here. In Java, I'd rather use:

```
Optional<Object> resolveName(String name);
```

which reads much better IMO. Indeed, for consistency I would rename that function to:

- `tryResolveName`